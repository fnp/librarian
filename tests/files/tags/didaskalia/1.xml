<naglowek_osoba>CZEŚNIK</naglowek_osoba>
<kwestia>

<didaskalia>obracając się ku niemu</didaskalia>

<strofa>
<wers_cd>Tu sęk właśnie!</wers_cd>
Na toż bym się, mocium panie,/
Kawalerstwa dziś wyrzekał,
</strofa>

<didaskalia>uderzając w stół</didaskalia>

<strofa>
By kto... niech go piorun trzaśnie!/
Długo będzie na to czekał,
</strofa>

<didaskalia>po krótkim milczeniu, biorąc talerz</didaskalia>

<strofa>
Ma dochody wprawdzie znaczne ---/
Podstolina ma znaczniejsze;/
Z wdówką zatem działać zacznę.
</strofa>

<didaskalia>po krótkim milczeniu</didaskalia>

<strofa>
Bawi z nami --- w domu Klary,/
Bo krewniaczka jej daleka,/
Ale mnie się wszystko zdaje...
</strofa>
</kwestia>
