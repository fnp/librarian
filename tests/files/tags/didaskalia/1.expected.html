<div id="book-text">
<h4 id="i1">CZEŚNIK</h4>
<div class="kwestia">
<div class="didaskalia" id="i2">obracając się ku niemu</div>
<div class="stanza" id="i3">
<a href="#f1" class="wl-num">1</a><div class="wl verse verse-cont" id="f1">Tu sęk właśnie!</div>
Na toż bym się, mocium panie,<div class="wl verse" id="f2">Kawalerstwa dziś wyrzekał,</div>
</div>
<div class="didaskalia" id="i4">uderzając w stół</div>
<div class="stanza" id="i5">
<div class="wl verse" id="f3">By kto... niech go piorun trzaśnie!</div>
<div class="wl verse" id="f4">Długo będzie na to czekał,</div>
</div>
<div class="didaskalia" id="i6">po krótkim milczeniu, biorąc talerz</div>
<div class="stanza" id="i7">
<a href="#f5" class="wl-num">5</a><div class="wl verse" id="f5">Ma dochody wprawdzie znaczne —</div>
<div class="wl verse" id="f6">Podstolina ma znaczniejsze;</div>
<div class="wl verse" id="f7">Z wdówką zatem działać zacznę.</div>
</div>
<div class="didaskalia" id="i8">po krótkim milczeniu</div>
<div class="stanza" id="i9">
<div class="wl verse" id="f8">Bawi z nami — w domu Klary,</div>
<div class="wl verse" id="f9">Bo krewniaczka jej daleka,</div>
<a href="#f10" class="wl-num">10</a><div class="wl verse" id="f10">Ale mnie się wszystko zdaje...</div>
</div>
</div>
</div>
